### DISCUSSION TITLE
Does Noobaa bucket have immutability

### DISCUSSION THREAD

**Discussion by vamseekrishna25** [noobaa/noobaa-core#6451] [Q&A]
can we set WORM(write once read many) type of objects in  buckets in noobaa

**Comment by nimrod-becker** [comment]
Hi @vamseekrishna25 

S3 WORM was added with https://github.com/noobaa/noobaa-core/pull/5976
However it was *not* extensively tested, and I would consider it more of a "beta" level, would appreciate feedback on that if you test and have some

**Reply by romayalon** [reply]
adding to Nimrod, you will need to set  WORM_ENABLED configuration in config.js 
`config.WORM_ENABLED = false;` to true.
